txpipe/lens_selector.py

```diff
@@ -312,9 +313,99 @@ def data_iterator(self):
 
 
 
-def flatten_list(lst):
-    return [item for sublist in lst for item in sublist]
 
+class TXLensCatalogSplitter(PipelineStage):
+    """
+    Split a lens catalog file into a new file with separate bins.
+    """
+    name='TXLensCatalogSplitter'
+
+    inputs = [
+        ('lens_tomography_catalog', TomographyCatalog),
+        ('photometry_catalog', HDFFile),
+    ]
+
+    outputs = [
+        ("calibrated_lens_catalog", HDFFile),
```

Comment:
is there any calibration for the lenses going on? should we call it something else?

Reply:
Renaming! Also we could change `calibrated_shear_catalog` to `binned_shear_catalog`.  See my conceptual poll on desc-3x2pt!